source/lambda/customResourceKendraDataSource/lambda_function.py

```diff
@@ -4,10 +4,10 @@
 def on_create(event, context):
```

Comment:
Could you please add the Amazon License Header to the file?
https://github.com/awslabs/document-understanding-solution/blob/fe6dbabcc983c31bc39fcf4dcf35bfabc8f1f2a9/source/lambda/documentbulkprocessor/lambda_function.py#L2